src/Extracting/Strategies/Responses/ResponseCalls.php

```diff
@@ -170,6 +174,13 @@ protected function runPreRequestHook(Request $request, ExtractedEndpointData $en
         }
     }
 
+    protected function runPostRequestHook(Request $request, ExtractedEndpointData $endpointData, JsonResponse $jsonResponse): void
```

Comment:
Is the response always of type JSONResponse? I don't think we should add that constraint.

Reply:
Oh yes, my bad, I fixed it